pkg/util/rand/rand.go

```diff
@@ -32,6 +32,16 @@ var rng = struct {
 	rand: rand.New(rand.NewSource(time.Now().UTC().UnixNano())),
 }
 
+func Int(max int) int {
+	if max <= 0 {
```

Comment:
by consolidate i mean (1) have stringn use intn (2) eliminate panicking in both.